Controllers/TimeTable.php

```diff
@@ -99,41 +85,74 @@ public function post(): Response
      */
     public function get(): Response
     {
-        // Filters for the sql select
-        $userIdForFilter = null;
         $searchTermForFilter = null;
         $now = CarbonImmutable::now();
         $ticketCacheExpiration = $this->settings->getSetting('itk-leantime-timetable.ticketCacheExpiration') ?? 1200;
 
-        if (isset($_GET['searchTerm'])) {
-            $searchTerm = $_GET['searchTerm'];
-        }
+        try {
```

Comment:
All this date handling should possibly, be handle with the data time helpers from https://github.com/Leantime/leantime/blob/master/app/Core/Support/DateTimeHelper.php 


Not sure about this, as I have lost overview over the date stuff.

Reply:
I have allowed certain cases for date get params, so it's kind of a custom solution. Im not sure much good will come out of utilizing the DateTimeHelper.